src/plugins/content_management/server/core/types.ts

```diff
@@ -30,9 +31,60 @@ export interface ContentStorage {
   delete(ctx: StorageContext, id: string, options: unknown): Promise<any>;
 }
 
+export interface RpcSchemas {
+  get?: {
+    in?: {
+      options?: Type<any>;
```

Comment:
Could it be `unknown` instead of `any` ? Or would it not work for registering a schema ?

Reply:
It would not work. `unknown` and `any` have different purposes. Here we are really saying it can be _any_ schema.